react/features/base/tracks/functions.js

```diff
@@ -200,6 +200,18 @@ export function getLocalVideoType(tracks) {
     return presenterTrack ? MEDIA_TYPE.PRESENTER : MEDIA_TYPE.VIDEO;
 }
 
+/**
+ * Returns the stored local video track.
+ *
+ * @param {Object} state - The redux state.
+ * @returns {Object}
+ */
+export function getLocalJitsiVideoTrack(state) {
+    const track = getLocalVideoTrack(state['features/base/tracks']);
+
+    return track?.jitsiTrack || null;
```

Comment:
```suggestion
    return track?.jitsiTrack;
```

Reply:
I don't think we need to explicitly return null here, do we? undefined will do.

Reply:
Fixed.